c-api/src/utils.rs

```diff
@@ -72,3 +72,15 @@ macro_rules! try_execute {
         }
     }};
 }
+
+#[macro_export]
```

Comment:
nit: `#[macro_export]` is not intended for internal macro helpers (if you remove it, I recommend putting all the helper macros in a `macros` module that must be declared before all the others, generally speaking macros must be declared before they are used).